sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2630,7 +2630,9 @@ object GpuOverrides {
       "The backend of the Scalar Pandas UDFs. Accelerates the data transfer between the" +
         " Java process and the Python process. It also supports scheduling GPU resources" +
         " for the Python process when enabled",
-      ExecChecks(TypeSig.commonCudfTypes, TypeSig.all),
+      ExecChecks(
+        TypeSig.commonCudfTypes + TypeSig.ARRAY.nested(TypeSig.commonCudfTypes + TypeSig.STRUCT),
```

Comment:
Same questions about the types supported here.

Reply:
Yes, the original goal is to support only array of struct for a customer request.

I have updated to support array of struct, array of array, top-level struct, struct of struct, struct of array, along with the integration tests, since cudf has already supported them.

It would be nice to have them here.